itests/eth_deploy_test.go

```diff
@@ -93,6 +87,15 @@ func TestDeployment(t *testing.T) {
 	pendingFilter, err := client.EthNewPendingTransactionFilter(ctx)
 	require.NoError(t, err)
 
+	// Disconnect peers so we can send messages without propegating them.
```

Comment:
This should work, but is it need easier to just pause and resume mining?

Reply:
I can do that?

Reply:
I can do that...

Reply:
Done.